### ISSUE TITLE
Parse error "Invalid value" when hosting Northstar server

### ISSUE THREAD

**Issue by GeckoEidechse** [R2Northstar/Northstar#685]
**TL;DR: Most likely port forwarding not set up correctly or limited by ISP, check [instructions on the wiki](https://r2northstar.gitbook.io/r2northstar-wiki/hosting-a-server-with-northstar/getting-started).**

Trying to host a gameserver one might see the following log entry:
```
[19:41:37] [NORTHSTAR] [errr] Failed reading masterserver authentication response: encountered parse error "Invalid value."
[19:41:37] [NORTHSTAR] [errr] Reached max ms server registration attempts
```

The reason behind this is that if [Atlas](https://github.com/R2Northstar/Atlas) cannot reach the gameserver that is trying to host it will send back a 5XX HTTP error response with appropriate error information in a JSON file.
For some reason, [Cloudflare](https://www.cloudflare.com/), the CDN we use for DDoS protection and caching, modifies this response on travel to add its own information to it.
This results in the client not being able to parse the resulting response and showing the unhelpful `Invalid value`  message instead.

To fix this, we need to figure out why Cloudflare is modifying the response and prevent it from doing so.
(There's probably a setting for this in the dashboard somewhere)
